devel/bazel/Portfile

```diff
@@ -230,6 +230,9 @@ build.env-append  CC=${configure.cc} \
               TMPDIR=${workpath}/tmp \
            JAVA_HOME=${java.home}
 
+# https://github.com/bazelbuild/bazel/issues/7940#issuecomment-553447868
+build.env-append      EXTRA_BAZEL_ARGS="--host_javabase=@local_jdk//:jdk"
+
```

Comment:
I would prefer if instead of appending this to the build env here, below where the build command is defined you only added it there, and also please only do this for the bazel-3.1 subport and not he others. Finally can you also please add a comment listing the trac ticket, instead of the upstream report.

Reply:
I tried that, but the build failed in the same way. I haven’t tracked it down but it appears that this must be an environment variable, not a shell variable (consistent with the issue comment). I’ll add this to `bazel-3.1` specifically.

Reply:
I haven’t checked, but I’m pretty certain that the other `BAZEL_EXTRA_ARGS` isn’t set correctly in the original Portfile.
The bazel build docs [say](https://docs.bazel.build/versions/0.23.0/install-compile-source.html) this must be an environment variable.

Reply:
Also, that same bazel documentation isn’t specific to version 3.1, so it’s likely this must be set for other versions as well, but I haven’t tracked them all down, only version 3.1.

Reply:
OK thanks. I've merged and will take a look myself.